src/org/labkey/test/params/FieldDefinition.java

```diff
@@ -323,7 +323,9 @@ public String toString()
         Flag("Flag", "Flag (String)", null),
         Attachment("Attachment", "Attachment", "attachment"),
         User("User", "User", "int"),
-        Lookup("Lookup", "Lookup", null),
+        Lookup("Lookup", "Lookup", null),   // Todo: convert usage to lookupTo<type>
+        LookupToInteger("Lookup", "Lookup", "Integer"),
+        LookupToString("Lookup", "Lookup", "String"),
```

Comment:
Unfortunately we have some usages where we use the columnType fields to do double-duty and that's why we still allow columnType to take a string.  Changing this will require time to shake out the failures and fix them.  I have a todo in ColumnType now, to coalesce these usages- I suggest we shake this out after 20.3